packages/blobs/riscv64/thead/lpi4a-sysfan.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+#ENABLE FAN (PWM1)
```

Comment:
Why not define this in dts?

Reply:
I will actively communicate with the LicheePi 4A kernel maintainer and submit a PR.

Reply:
What I mean is that I will submit a PR to the corresponding kernel repository of the device to make the change.

Reply:
The kernel device tree used by this device already contains a PWM fan.